sql/executor.go

```diff
@@ -83,6 +88,7 @@ func (e Executor) Execute(args driver.Request) (driver.Response, int, error) {
 	if planMaker.txn != nil {
 		planMaker.session.Txn = &planMaker.txn.Proto
 		planMaker.session.MutatesSystemDB = planMaker.txn.SystemDBTrigger()
+		planMaker.session.Timestamp = driver.Timestamp(txnTimestamp)
```

Comment:
we talked and we decided to create a new message that embeds proto.Transaction and adds a sql transaction timestamp.